issue found on: server/acks_utils.go:64:1: paramTypeCombine

an issue has been detected in source code: ./server/acks_utils.go:64:1: paramTypeCombine: func(ruleSelector types.RuleSelector, justification string, createdBy string) could be replaced with func(ruleSelector types.RuleSelector, justification, createdBy string)

--- a/server/acks_utils.go
+++ b/server/acks_utils.go
@@ -61,7 +61,7 @@ func handleMissingRule(writer http.ResponseWriter, ruleSelector string) {
 }
 
 // addNewRule function add a new rule to existing map of acknowledges.
-func addNewRule(ruleSelector types.RuleSelector, justification string, createdBy string) {
+func addNewRule(ruleSelector types.RuleSelector, justification, createdBy string) {
 	// add new rule
 	acks[ruleSelector] = types.Acknowledge{
 		Acknowledged:  true,

